dbus/methods.go

```diff
@@ -598,3 +598,43 @@ func unitPath(name string) dbus.ObjectPath {
 func unitName(dpath dbus.ObjectPath) string {
 	return pathBusUnescape(path.Base(string(dpath)))
 }
+
+type JobStatus struct {
+	Id       int
+	Unit     string
+	JobType  string
+	Status   string
+	JobPath  string
+	UnitPath string
```

Comment:
This and the one above are `ObjectPath`.

Reply:
Done